lua/carbon-now.lua

```diff
@@ -1,28 +1,11 @@
 -- module represents a lua module for the plugin
+local default_config = require("config.default")
 local carbon = {}
 
--- default configs
-carbon.config = {
```

Comment:
why create another file for this?

Reply:
Hi @ellisonleao, i came as fast as possible, from my work, so. I saw this pattern a lot, in other lua scripts/plugins, like `nvim-cmp` or `gitsigns.nvim`, where the default configuration is abstracted into it's own file...  having a separated `default.lua` file to define the default configuration for your Neovim plugin is a good practice. It promotes code organization, reusability, extensibility, and allows future users to customize the plugin to their needs on demand.

This way, you can require only the configuration, without any bias to mutate the result or run any extra code along the way...

So let's do a brief (long) Recap of what i think:
1. Separation of Concerns: By abstracting the default configuration into a separate file, you're keeping the configuration logic isolated from the rest of the plugin code. This promotes a modular & organized structure making it easier to maintain and understand.

2. Reusability: Having the default configuration in its own file makes it reusable across different parts of your plugin.If there are multiple components or modules that require access to the default configuration, they can simply import the default.lua file and use the default_config table. (Actually, that was the main reason here, when i include this default settings on your tests)

3. Extensibility: Separating the default configuration allows for easy extensibility in the future. If you decide to introduce additional configuration options or modify existing ones, you can do so by updating the default.lua file without having to modify other parts of your plugin code, creating different flavors of it,

Reply:
hey @Domenic-MZS those are good and valid points, but this is a very small plugin and i don't see the need for it. Can you please just leave as it is now please?

Reply:
Okay, will fix it in the next commit

Reply:
Fixed with [8aa5de1](https://github.com/ellisonleao/carbon-now.nvim/pull/4/commits/8aa5de1d050c41f69bdedd20aebaff80a4f43313)